mstflint: fails to build with NLS

### Package Name

mstflint

### Maintainer

@tk154

### OpenWrt Version

snapshot and OpenWrt 24.10

### OpenWrt Target/Subtarget

mvebu/cortex-a9 (Turris Omnia)

### Steps to Reproduce

1. Enable CONFIG_BUILD_NLS
2. Try to compile package

### Actual Behaviour

It fails with following output:
```
checking for xmlDocDumpFormatMemoryEnc in -lxml2... no
configure: error: cannot find xml2 xmlDocDumpFormatMemoryEnc() function. Try re-installing the library...
make[2]: *** [Makefile:145: /Volumes/OpenWrt/omnia-hbd/build/build_dir/target-arm_cortex-a9+vfpv3-d16_musl_eabi/mstflint-4.29.0/.configured_68b329da9893e34099c7d8ad5cb9c940] Error 1
```

I was able to fix this by adding this row to Makefile:
```
include $(INCLUDE_DIR)/nls.mk
```

### Confirmation Checklist

- [x] The package is maintained in this repository.
- [x] I understand that issues related to [the base OpenWrt repository](https://github.com/openwrt/openwrt/issues) or [LuCI repository](https://github.com/openwrt/luci/issues) will be closed.
- [x] I am reporting an issue for OpenWrt, not an unsupported fork.

--- a/utils/mstflint/Makefile
+++ b/utils/mstflint/Makefile
@@ -25,6 +25,7 @@ PKG_BUILD_PARALLEL:=1
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
 
 define Package/mstflint
   SECTION:=Utilities

